app/uk/gov/hmrc/cdsreimbursementclaimfrontend/models/Claim.scala

```diff
@@ -31,8 +32,8 @@ final case class Claim(
 )
 
 object Claim {
-  implicit class ListClaimOps(private val claims: List[Claim]) {
-    def total: BigDecimal = claims.map(c => c.claimAmount).sum
+  implicit class ListClaimOps(private val claims: NonEmptyList[Claim]) {
```

Comment:
`private val` is not needed for regular classes like in `class ListClaimOps`, in this case member been encapsulated bu the default.
Things were different for case class `case class ListClaimOps[A](value: A)` where value will be public by the default.

Reply:
Done